crates/libs/bindgen/Cargo.toml

```diff
@@ -9,6 +9,9 @@ description = "Windows metadata compiler"
 repository = "https://github.com/microsoft/windows-rs"
 readme = "readme.md"
 
+[lib]
+crate-type = ["rlib", "dylib"]
```

Comment:
What's this for?

Reply:
Improves build times, by allowing linking `windows-bindgen` to a DLL and reusing the DLL, rather than linking it into every tool that uses it.  This cut about 2s off my build times.